### ISSUE TITLE
org.freedesktop.portal.Desktop does not support OpenURI method

### ISSUE THREAD

**Issue by OldNick** [tiliado/nuvolaplayer#714]
I'm getting this after a fresh install on openSuse Leap 15.2 (Gecko), Plasma desktop. (The script is for BBC Iplayer)
Having to use console to start as there is no icon for the app.

Console output :
`n@r:~>flatpak run eu.tiliado.NuvolaAppBbcIplayer

Runner: [WARNING Δ605846us Nuvola] StartupWindow.vala:146: xdg-desktop-portal-message: org.freedesktop.portal.Desktop does not support OpenURI method of org.freedesktop.portal.OpenURI interface.
Your system is broken: dlsym doesn't work! This has been reported to be caused by Nvidia's libGL. You should expect time related functions to misbehave. https://bugs.chromium.org/p/chromium/issues/detail?id=16800
[0309/194552.852821:ERROR:sandbox_linux.cc(368)] InitializeSandbox() called with multiple threads in process gpu-process.`

Possibly a package is missing in the openSuse install?

**Comment by fenryxo** [comment]
Hello @OldNick, could you try `zypper install xdg-desktop-portal xdg-desktop-portal-kde`? As for the missing icon for the app, did you log out and log in again after the installation of the BBC iPlayer app?

**Comment by OldNick** [comment]
I used Yast...
xdg-desktop-portal-kde was the missing element - and with it came the icon! It wasn't installed as a dependency with xdg-desktop-portal - I noticed there's also xdg-desktop-portal-gdk which may be necessary for the gnome flavour of OpenSuse

The BBC are shutting down all streaming to 3rd party apps (I've been using radiotray until now) and directing everyone to their website. With luck your solution will future-proof us from any more of their decisions - expect an increase in custom!
